src/views/emails/welcome.html.ejs

```diff
@@ -0,0 +1,125 @@
+<html lang="en">
```

Comment:
[handlebars](https://sendgrid.com/docs/for-developers/sending-email/using-handlebars/) could be nice to have here